sdk/documentintelligence/azure-ai-documentintelligence/azure/ai/documentintelligence/_operations/_patch.py

```diff
@@ -266,7 +321,361 @@ def get_long_running_output(pipeline_response):
+        """
+        _headers = case_insensitive_dict(kwargs.pop("headers", {}) or {})
+        _params = kwargs.pop("params", {}) or {}
+
+        content_type: Optional[str] = kwargs.pop("content_type", _headers.pop("content-type", None))
+        cls: ClsType[_models.AnalyzeResult] = kwargs.pop("cls", None)
+        polling: Union[bool, AnalyzeDocumentLROPollingMethod] = kwargs.pop("polling", True)
+        lro_delay = kwargs.pop("polling_interval", self._config.polling_interval)
+        cont_token: Optional[str] = kwargs.pop("continuation_token", None)
+        if cont_token is None:
+            raw_result = self._analyze_document_initial(
+                model_id=model_id,
+                analyze_request=analyze_request,
+                pages=pages,
+                locale=locale,
+                string_index_type=string_index_type,
+                features=features,
+                query_fields=query_fields,
+                output_content_format=output_content_format,
+                output=output,
+                content_type=content_type,
+                cls=lambda x, y, z: x,
+                headers=_headers,
+                params=_params,
+                **kwargs,
+            )
+            raw_result.http_response.read()  # type: ignore
+        kwargs.pop("error_map", None)
+
+        def get_long_running_output(pipeline_response):
+            response_headers = {}
+            response = pipeline_response.http_response
+            response_headers["Retry-After"] = self._deserialize("int", response.headers.get("Retry-After"))
+            response_headers["Operation-Location"] = self._deserialize(
+                "str", response.headers.get("Operation-Location")
+            )
+
+            deserialized = _deserialize(_models.AnalyzeResult, response.json().get("analyzeResult"))
+            if cls:
+                return cls(pipeline_response, deserialized, response_headers)  # type: ignore
+            return deserialized
+
+        path_format_arguments = {
+            "endpoint": self._serialize.url("self._config.endpoint", self._config.endpoint, "str", skip_quote=True),
+        }
+
+        if polling is True:
+            polling_method = AnalyzeDocumentLROPollingMethod(
+                lro_delay, path_format_arguments=path_format_arguments, **kwargs
+            )
+        elif polling is False:
+            polling_method = cast(AnalyzeDocumentLROPollingMethod, NoPolling())
+        else:
+            polling_method = polling
+        if cont_token:
+            return cast(
+                AnalyzeDocumentLROPoller, LROPoller[_models.AnalyzeResult].from_continuation_token(
+                    polling_method=polling_method,
+                    continuation_token=cont_token,
+                    client=self._client,
+                    deserialization_callback=get_long_running_output,
+                )
+            )
+        return AnalyzeDocumentLROPoller[_models.AnalyzeResult](
```

Comment:
Can you open an issue on autorest.python for a feature request to be able to generate with custom pollers / polling methods?

Reply:
Logged issue: https://github.com/Azure/autorest.python/issues/2816